src/sass/_license-area.scss

```diff
@@ -0,0 +1,9 @@
+//
+// Footer within tabs
+// --------------------------------------------------
+
+.tab footer .license-area {
```

Comment:
Have you tried if the selector works without footer? Could make it harder to change the mark-up in the future.